modules/upgrade/pages/db-migration.adoc

```diff
@@ -1,61 +1,65 @@
 [[db-migration]]
 = Database Migration
 
+This is about PostgreSQL database migration.
 
-When you installed {productname} Server 3.1 or earlier on {sls}{nbsp}12{nbsp}SP2 or earlier, it was based on a PostgreSQL{nbsp}9.4 database.
-PostgreSQL{nbsp}9.4 can be upgraded to PostgreSQL{nbsp}9.6.
+If you want to upgrade {productname} 3.2 to {productname} 4 you must check whether {productname} 3.2 is already based PostgreSQL version 10.
```

Comment:
```suggestion
If you want to upgrade {productname} 3.2 to {productname} 4 you must check whether {productname} 3.2 is already based on PostgreSQL version 10.
```

Reply:
I'm not a native English speaker but I always thought that "must" is considered impolite and one should use "you will need to" instead. But if it has Lana's blessing it will be ok :)

Reply:
@mantel This is technical documentation (not marketing speech), Hubert!
We probably should define somewhere what it actually means if we say "must" or "should".